src/chain/context-runner-impl.ts

```diff
@@ -43,14 +44,19 @@ export class ContextRunnerImpl implements ContextRunner {
             path,
           });
 
-          // An instance is mutable, so if an item changed its value, there's no need to call getData again
-          const newValue = instance.value;
+          if (contextItem instanceof Rename) {
+            // change instance path
+            instance.path = contextItem.newPath;
```

Comment:
Is it correct to change `instance.path`?

Reply:
I'll guess _no_ because of the below:
https://github.com/express-validator/express-validator/blob/6715a6a10963ef29f0eaf3c9e25463187afeab99/src/context.ts#L66-L70

but we can always change behaviour.

And side comment, I'd just find it a bit more classy if we could use context's APIs instead of checking for the specific implementation of `Rename` here.